packages/api/src/configurations/apis.conf.ts

```diff
@@ -5,3 +5,5 @@ export const API_SENDINBLUE_TOKEN = process.env.API_SENDINBLUE_TOKEN;
 
 export const DAUPHIN_USERNAME = process.env.DAUPHIN_USERNAME;
 export const DAUPHIN_PASSWORD = process.env.DAUPHIN_PASSWORD;
+
+export const DEMARCHES_SIMPLIFIEES_TOKEN = process.env.DEMARCHES_SIMPLIFIEES_TOKEN;
```

Comment:
Je pense qu'on peut renommer en "DS" pour réduire la taille de la variable. On utilise toujours cette abréviation donc je pense pas qu'on soit perdu un jour. Au pire on rajoute un commentaire au dessus. @VSerain qu'en penses-tu ?

`process.env.DS_API_TOKEN`

Reply:
Moi je confuse entre Démarches Simplifiées et Data Subventions ^^

Reply:
Ba à l'époque où l'on avait choisi le nom du produit, j'avais dit qu'il fallait faire gaffe parce que qu'on s'appelait aussi DS mais ça n'avais l'air de choquer personnes ! 
Cella dit DS c'est très clair chez tous le monde que c'est démarche simplifiée !
Après, je me dis qu'en même temps, on paye pas aux caractères et que c'est pas une variable qu'on va écrire 1 00 000 fois donc why not de le laisser comme ça.

Reply:
Ça nous fait pas trop trancher, ça ^^

Reply:
On garde démarche simplifiée !

Reply:
ého ! t'es pas le chef à valider comme ça @VSerain :p

Mais je vais pas chipoter pour ça en effet

Reply:
Alors non, je suis pas chef (Sauf sur overcook mais c'est un autre sujet) !  Mais du coup ça faisait 2 pour Démarche Simplifier et un pour DS ;)